cli/login/login.go

```diff
@@ -0,0 +1,101 @@
+package login
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/buildbuddy-io/buildbuddy/cli/arg"
+	"github.com/buildbuddy-io/buildbuddy/cli/log"
+	"github.com/buildbuddy-io/buildbuddy/cli/storage"
+)
+
+const (
+	apiKeyFileName = "apikey"
+	apiKeyHeader   = "remote_header=x-buildbuddy-api-key"
```

Comment:
nit: this might not work later if we make the arg parsing more robust - might save some headache later if we have a util func in the arg pkg like GetRemoteHeaders() that returns a map[string]string. Or a TODO so we don't forget

Reply:
Good catch, I've added a TODO. We might squeak by if we go the arg normalization / canonicalization route.